docs/current/guides/snippets/use-services/expose-service-containers-to-host/main.go

```diff
@@ -0,0 +1,59 @@
+package main
```

Comment:
@helderco @TomChv @slumbering I have attempted to write equivalent code for this snippet in Node and Python but could not test or debug it yet. Could you please have a look and modify it as needed?